src/events/unused-eventscript.h

```diff
@@ -17,7 +17,7 @@ CONST_DATA EventListScr EventScr_UnkCh3B_EndingScene[] = {
     ENDA
 };
 
-CONST_DATA u8 gAutoUdefJids[8] = {
+CONST_DATA u8 gAutoUdefJids[2][4] = {
```

Comment:
If you don't mind, could you also update the contents of the array to use the constants defined in `include/constants/classes.h`?